MetadataExtractor/DirectoryExtensions.cs

```diff
@@ -348,6 +348,10 @@ public static byte[] GetByteArray(this Directory directory, int tagType)
             if (o == null)
                 return null;
 
+            var strvalue = o as StringValue;
+            if (strvalue != null)
+                return strvalue.Bytes;
+
             byte[] bytes;
 
             var rationals = o as Rational[];
```

Comment:
I might leave that decision to you. Your experience with how people use these methods far exceeds mine. But, it seems like the two overloads of GetString() could be combined into one if you wanted to really enforce this. That would at least shorten the number of overloads.

I'm not so sure on a GetStringValue version -- unless it ended up calling a combined GetString() somehow. I think you might answer your own question there at the end; it doesn't seem to make much sense to hold a double in a byte[] in a class with 'String' in its name.